fr/intro/conventions.rst

```diff
@@ -43,6 +43,16 @@ la convention est que vos URLs soient en minuscules avec des tirets en utilisant
 la classe ``DashedRoute``, donc ``/red-apples/go-pick`` est la bonne forme
 pour accéder à l'action ``RedApplesController::goPick()``.
 
+Quand vous créez des liens en utilisant ``this->Html->link()``, vous pouvez
+utiliser les conventions suivantes pour le tableau d'url::
+
+    $this->Html->link('link-title', [
+        'prefix' => 'my-prefix // unmodified
```

Comment:
I think MyPrefix makes sense as that's the name of the namespace. Looking at InflectedRoute and the Default route class, neither of those inflect prefix - which is also odd.